tests/e2e/scenarios/test_reborn_webui_v2_smoke.py

```
@@ -204,6 +210,63 @@ async def _wait_for_automation_named(
+                await asyncio.sleep(0.5)
+    except TimeoutError:
+        if absent:
+            expectation = "to be absent"
+        elif expected_state is not None:
+            expectation = f"to have state {expected_state!r}"
+        else:
+            expectation = "to be present"
+        raise AssertionError(
+            f"Timed out waiting for automation {automation_id!r} {expectation}. "
+            f"Last body: {last_body}"
+        ) from None
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)